runtime-scientific/ucx/autobuild/defines

```diff
@@ -0,0 +1,26 @@
+PKGNAME=ucx
+PKGSEC=libs
+PKGDEP="rdma-core fuse-3"
+BUILDDEP__AMD64="${BUILDDEP} cuda valgrind"
+BUILDDEP__ARM64="${BUILDDEP} cuda valgrind"
+PKGDES="A communication framework for high-bandwidth and low-latency networks"
+PKGSUG="cuda"
```

Comment:
What happens if CUDA is not installed? Does the user get an error?

Reply:
No. CUDA will not be used unless it is explicitly invoked.